apps/dashboard/src/app/team/[team_slug]/[project_slug]/insight/components/BlueprintsPageHeader.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { PlusIcon } from "lucide-react";
+
+export function BlueprintsPageHeader() {
+  return (
+    <div className="border-b pb-4">
+      <div className="flex flex-col gap-4 md:pb-4 lg:flex-row lg:justify-between">
+        <div>
+          <h1 className="mb-1.5 font-semibold text-3xl tracking-tight lg:text-4xl">
+            Blueprints
+          </h1>
+        </div>
+        <Button className="cursor-not-allowed gap-2 opacity-50" disabled>
+          <div className="flex items-center">
```

Comment:
Remove this `div` to fix the spacing between icon and text

Reply:
done